src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.tsx

```diff
@@ -54,6 +60,31 @@ const FilterList = styled(List)(({ theme }) => ({
   },
 }));
 
+type ShowMenuItemProps = {
+  item: NavItems;
+  user: Session['user'];
+  hasOrganizationsAccess: boolean;
+};
+
+const showMenuItem = ({
+  item,
+  user,
+  hasOrganizationsAccess,
+}: ShowMenuItemProps): boolean => {
+  if (item?.grantedAccess?.length) {
+    if (hasOrganizationsAccess && ['organizations'].includes(item.id)) {
```

Comment:
Is there a reason we're not doing this?

```suggestion
    if (hasOrganizationsAccess && item.id === 'organizations')) {
```

Reply:
Yes, as there are 3 organization links `organizations`, `organizations/accountLists` & `organizations/contacts'`

Reply:
In that case, don't you want `item.id.includes('organizations')` or `item.id.startsWith('organizations')`? What you have right how will only match 'organizations'.

Reply:
You're right, it's weird that the links were showing